jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/MultiPartFormFields.java

```diff
@@ -0,0 +1,76 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.server;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.http.MimeTypes;
+import org.eclipse.jetty.http.MultiPart;
+import org.eclipse.jetty.http.MultiPartFormData;
+import org.eclipse.jetty.io.Content;
+import org.eclipse.jetty.util.Attributes;
+
+public class MultiPartFormFields
```

Comment:
Not sure I understand this class.
It seems exactly identical to using the parser directly, and it can only be used on the server, which seems an unnecessary restriction?
If it saves a couple of lines of code, then we should improve `MultiPartFormData`, rather than introducing more server-only classes.

Reply:
The static method taking a `Content.Source` can just just be on `MultiPartFormData`?
I don't know if we need the one taking a `Request`

Reply:
@gregw I do not understand the difference between `MultiPartFormData` and `MultiPartFormFields`.

The latter seems a utility class to save perhaps a couple of lines of code, but introducing 2 more, server-only, classes.